Desktop version

ommarz opened the pull request:
Setup desktop version
Fix mobile version due to changes for desktop version

ommarz committed:
[Fix] [desktop version styling]

[Add]images for icons and background
[Fix styling] [toolbar, headline, works]

ommarz committed:
[Add] [div tag]

ommarz committed:
[Fix] [styling]

ommarz committed:
[Fix] [grid template]

Improvements in styling for works section

ommarz committed:
[Add] [divs in list for skills]

Improvements to styling for desktop version
Add divs to elements in list in order to change positioning in desktop
version
This might change positioning in mobile verison.

ommarz committed:
[Fix] [styling in multiple cards in works section]

Improvements in styling to meet template. Need as changes made for
desktop version changed mobile version.
Setup correct margin between cards.

ommarz committed:
[Remove] [border in projects used as reference]

ommarz committed:
[Fix] [list styling in about_me section]

Bug fix created while coding desktop version
Center correctly skills list

ommarz committed:
[Add] [Framework element for desktop version]

Improvement to meet template

ommarz committed:
[Fix] [Styling in about me section list desktop]

Improvements to meet template
Fix positioning of list elements

ommarz committed:
[Add] [fields in form]

Improvements to meet template requirements
Add fields missing for desktop version, add class desktop to elements
that will appear in said version
Add class mobile to elements that will appear in that version

ommarz committed:
[Fix] [styling to contact me section]

Improvements to setup the section as in template
Fix styling in contact intro and contact form

ommarz committed:
[Fix] [linters]

KanzaTahreem commented on stylesheet.css:
```diff
@@ -585,3 +618,537 @@ input {
+    background-image: url('./images/background/headline-background-desktop.svg');
+    background-repeat: no-repeat;
+    background-size: 50% 90%;
+  }
```
- [x] Good job so far 👍👍 Please make sure that the background image is placed according to the Figma template to get a design close to the Figma template as much as possible 📌✨



Your design | Given design
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/98db6eb2-0fbb-497e-b91e-908c55a8b3ee) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/507775ae-43f7-46fb-831e-fab783438c05)

KanzaTahreem commented on stylesheet.css:
```diff
@@ -585,3 +618,537 @@ input {
+    max-width: fit-content;
+    padding-left: 5vw;
+  }
+
```
- [x] I think it would be better to remove/reduce the `padding-left` here so that there is less space between the icons and content and a bit more space between background image and content as given in the design guidelines 💪 What do you think about this? 😀

Your design | Given design
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/284a3e19-7404-474c-a35b-59eead916075) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/2c426659-bc04-41ea-9dea-8e354b3261dd)

- [x] Similarly, kindly reduce the horizontal space from the following for getting an almost exact design as given in the design guidelines as mentioned in the project requirements

Your design | Given design
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/70f275d7-522a-4353-802d-a686cd293dda) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/76102172-781e-48af-83a6-efe4288aa7a4)

KanzaTahreem commented on stylesheet.css:
```diff
@@ -585,3 +618,537 @@ input {
+    min-width: 90vw;
+    align-items: flex-start;
+    justify-content: space-between;
+  }
```
- [x] Could you please increase the space between the line and main project and reduce the space between list items😀 for getting a design close to the figma template as much as possible ⛏✨
- [x] Also, please make sure to horizontally to this section ✌
- [x] Kindly style the button following the Figma template 📌

Your design | Given design
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/a47d4a6d-c3b0-4c75-9580-47a9e34bf587) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/1b5b04a2-29d0-4078-818e-3b9715274d12)

KanzaTahreem commented on stylesheet.css:
```diff
@@ -585,3 +618,537 @@ input {
+    padding: 0;
+    align-items: flex-start;
+    justify-content: flex-end;
+  }
```
- [x] Good job with the project cards 🙌However, please make sure to adjust the `width` and `height` of the cards so that  we have a design close to the Figma template as mentioned in the project guidelines
- [x] Also, please make sure to add a bit more space horizontally and increase the padding of the list items to match the given design ⛏🤓
- [ ] Right now, when I am hovering over the project card, nothing is happening. If you look closely at the first card in the Figma template, it has a different cursor, which usually reflects the hover effect. Therefore, please make sure that when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image 🤓⛏ You could simply do this by applying same styles to all cards (except main project) and then apply the styles of the first project as `hover` for the cards


Your design | Given design
---| ---| 
![image](https://github.com/ommarz/Portfolio/assets/106119105/7722c965-7ff1-4d92-ada8-121d2259730f) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/4008d273-20fb-495c-9039-1af9c4eb28e6)

KanzaTahreem commented on stylesheet.css:
```diff
@@ -585,3 +618,537 @@ input {
+    background-position: 100% 20%;
+    background-size: 50vw;
+    align-items: flex-start;
+  }
```
- [x] The background image in the about section is overlapping with the line. Kindly fix this 💪
- [x] Also please make sure that the text and the line is aligned as given in the design guidelines 🙌✨
- [x] The background color for the about section is `background: #262626;`, could you please fix this? 😀

  > _You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._

Your design | Given design
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/a31e8758-1fdb-4924-825d-16c8beebe07e) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/8ef71ddb-0289-45c4-bbec-ade67cec345b)

KanzaTahreem commented on stylesheet.css:
```diff
@@ -585,3 +618,537 @@ input {
+    justify-content: space-between;
+    margin: 10vh 0 15vh 0;
+  }
+
```
- [x] Similarly here, please make sure to add a bit more space horizontally for getting a design close to the Figma template as much as possible  💪

Your design | Given design 
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/00bdb485-fa49-476b-94ba-d995ab7e4dca) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/c139e52c-10ac-4c48-9678-172cbe058150)

KanzaTahreem commented on index.html:
```diff
@@ -140,40 +150,52 @@ <h2 class="about_me_title">About me</h2>
+            required class="mobile"></label>
+        <label for="name"><input id="name" type="text" name="name" placeholder="First Name" maxlength="30"
+            required class="desktop"></label>
+        <label for="last_name"><input id="last_name" type="text" name="last_name" placeholder="Last Name" maxlength="30"
```
- [x] Good job for adding two input fields for name 💪, however as mentioned in the project requirements,  please make sure to use only one field for name 

  > _IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions._

- [x] I think it would be better to reduce the space between the footer icons ✨📌for getting an almost exact replica of the given design 

Your design | Given design
---| ---|
![image](https://github.com/ommarz/Portfolio/assets/106119105/846f0fbb-eaea-4869-9470-42c198f27083) | ![image](https://github.com/ommarz/Portfolio/assets/106119105/b03708dd-444f-4717-9840-815143ff363f)

KanzaTahreem reviewed (changes requested):
# Changes Required 🔁

Hi @ommarz 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

![so close](https://media.tenor.com/3sCHnD_x2RIAAAAC/almost-there.gif)


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

- [x] Kindly make sure to apply the `active` interaction for all of your buttons 💪✨Also, couls you please add a `border` for the `hover` interaction ? 🙌

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ommarz committed:
[Fix] [styling in buttons hover and active]

Improvements to meet template

ommarz committed:
[Fix] [background positioning in headline]

ommarz committed:
[Fix] [left-padding in content div]

ommarz committed:
[Fix] [header horizontal distance]

ommarz committed:
[Fix] [styling in recent works section]

Improvements to meet template requirements
[Fix] horizonal spacing
[Fix] spacing between element

ommarz committed:
[Fix] [About me section background positioning]

Improvements to meet template
Changed position from 20% to 15%

ommarz committed:
[Fix] [about me break width]

Imrpovements to meet template
Changed hr element size. Flex automatically  fixed start position

ommarz committed:
[Fix] [About Me section background color]

Improvements to meet template requirements
Changed color to #262626

ommarz committed:
[Fix] [skills list width]

Imrpovements to meet template requirements
Change width to 80vw

ommarz committed:
[Fix] [form inouts]

Fix to meet project requirements
Remove name and last name inputs

ommarz committed:
[Fix] [socials footer positioning]

Improvements to meet template requirements
Change width of list elements and flex

ommarz committed:
Merge branch 'main' of https://github.com/ommarz/Portfolio into desktop_version

ommarz committed:
[Fix] [linters]

OyePriscilla commented on stylesheet.css:
```diff
@@ -594,3 +632,553 @@ textarea {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 90vh;
+    width: 100vw;
+    background-image: url('./images/background/headline-background-desktop.svg');
+    background-repeat: no-repeat;
+    background-size: 50% 100%;
+  }
```
As raised earlier, kindly ensure that the background image is placed properly as we have it in figma. Though, I noticed that it worked fine below the screen size of 905px, I guess the relative unit you used for the `backround-size` is interfering. ℹ️ 

<img width="561" alt="Untitled" src="https://github.com/ommarz/Portfolio/assets/103386183/1b115f57-3ddb-4bc3-81ad-5e86107ee560">

OyePriscilla commented on stylesheet.css:
```diff
@@ -594,3 +632,553 @@ textarea {
+    height: fit-content;
+  }
+
+  .project_card {
+    display: flex;
+    flex-direction: column;
+    width: 26vw;
+    height: 50vh;
+    margin-bottom: 2vh;
+    padding: 0;
+    align-items: flex-start;
+    justify-content: flex-end;
+  }
```
Please kindly implement hover effect for the project cards as suggested earlier. When a user hover around the card the design below supposed to pop-up (project image and the see project button); by default the project button you added should not be there. 👇 

<img width="138" alt="Untitled" src="https://github.com/ommarz/Portfolio/assets/103386183/157f2543-f000-4698-bdb5-006032e75a7b">

OyePriscilla reviewed (changes requested):
Hi @ommarz 👏 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425724-8ca5825c-b02a-4983-b976-4bd0a2f70f9a.gif)

 ## Highlights
Implemented responsive design ✔️ 

 ## Required Changes ♻️
Kindly check the comments under the review, kindly pay close attention to the given template and reproduce it as close as posible. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ommarz committed:
[Add] [card 2 and card3 background images]

ommarz committed:
[Fix] [styling in project cards]

Improvements in styling to meet template
Changed styling while cards inactive and when hover over them

ommarz committed:
[Fix] [headline section background  positioning]

taldr27 reviewed (approved):
# Status: Approved! 🟢 
Hi @ommarz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

![congrats](https://media.giphy.com/media/3o6fIUZTTDl0IDjbZS/giphy.gif)

## Highlights 🎉 
✅ Professional and customized README. 🤓
✅ Linters tests passing.🤓
✅ Good use of media queries.🙌
✅ Great work with your design.💯
✅ Incorporates feedback from previous code reviews.🎯

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_
